Packs/CommonScripts/Scripts/ListUsedDockersImages/ListUsedDockersImages.yml

```diff
@@ -0,0 +1,35 @@
+args:
+- default: false
+  description: Export result to context
+  isArray: false
+  name: export_to_context
+  required: false
+  secret: false
+  auto: PREDEFINED
+  predefined:
+     - "true"
+     - "false"
+  defaultValue: "true"
+comment: 'List all dockers images that are in use by the installed integrations and automations'
+commonfields:
+  id: ListUsedDockersImages
+  version: -1
+enabled: false
+name: ListUsedDockersImages
+outputs:
+- contextPath: GetUserDockersImages.Output
+  description: 'Table of dockers images that are in use by the XSOAR system'
+  type: String
```

Comment:
this is not the updated context outputs.
You can use the new `demisto-sdk generate-context` command to generate the context outputs here.